scripts/eval/yamls/hf_eval.yaml

```diff
@@ -37,11 +37,11 @@ models:
 device_eval_batch_size: 4
 
 # FSDP config for model sharding
-fsdp_config:
-  sharding_strategy: FULL_SHARD
-  mixed_precision: FULL
-  forward_prefetch: True
-  limit_all_gathers: True
+# fsdp_config:
```

Comment:
I thought the previous change here was intentional, was it not?

Reply:
It was intentional, this was a mistake. w/ mixed init, fsdp is required